common/src/misc.rs

```diff
@@ -79,3 +82,304 @@ impl OutboundLocalTrust {
         self.outbound_trust_scores.insert(peer_id, value);
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct LocalTrustStateResponse {
+    result: Vec<(String, String, f32)>,
+    next_token: Option<String>,
+}
+
+impl LocalTrustStateResponse {
+    pub fn new(result: Vec<(String, String, f32)>, next_token: Option<String>) -> Self {
+        Self { result, next_token }
+    }
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct SeedTrustStateResponse {
+    result: Vec<(String, f32)>,
```

Comment:
Also here, we can use SeedEntry

Reply:
Done https://github.com/openrankprotocol/openrank/pull/220/commits/8e27fcf454bf645504b1b461614989be34673128